packages/wrangler/src/d1/create.tsx

```diff
@@ -24,18 +24,32 @@ export function Options(yargs: CommonYargsArgv) {
 			describe:
 				"A hint for the primary location of the new DB. Options:\nweur: Western Europe\neeur: Eastern Europe\napac: Asia Pacific\nwnam: Western North America\nenam: Eastern North America \n",
 			type: "string",
-			choices: LOCATION_CHOICES,
+		})
+		.option("experimental", {
```

Comment:
How about using `experimental-backend`, similar to how in `wrangler dev` we had `experimental-local`?
That would keep this flag tightly bound to its purpose and allow you to have other experimental flags in the future.